provider/cmd/pulumi-resource-command/schema.json

```diff
@@ -482,7 +482,7 @@
         },
         "localPath": {
           "type": "string",
```

Comment:
Before we GA, should we transition this to an asset/archive?

Reply:
Good call, especially given #42